.travis.yml

```diff
@@ -1,2 +1,9 @@
 language: ruby
+services:
+  - rabbitmq
+before_script:
```

Comment:
We can very easily achieve that by passing`[--exclude-pattern](https://www.relishapp.com/rspec/rspec-core/v/3-1/docs/configuration/exclude-pattern)` to rspec. But we'll have to update the rspec dependency to at least `3.1`. Does that sound fine?